packages/payment-integration-api/src/payment/storefront-payment-request-sender.ts

```diff
@@ -0,0 +1,24 @@
+import { RequestSender } from '@bigcommerce/request-sender';
+
+import { ContentType, INTERNAL_USE_ONLY, SDK_VERSION_HEADERS } from '../common/http-request';
+
+export default class StorefrontPaymentRequestSender {
```

Comment:
Is this implemented by other providers as well?

Reply:
Yes, quadpay and opy use it too

Reply:
Thanks, I feel this is something we can just do directly in the payment strategy. Do we need to expose a shared method here?